e2e/test-component/scenarios/embedding-sdk/tooltip-reproductions.cy.spec.tsx

```diff
@@ -64,7 +64,11 @@ describeEE("scenarios > embedding-sdk > tooltip-reproductions", () => {
     testCases.forEach(zIndexTestCase => {
       cy.get("@dashboardId").then(dashboardId => {
         mountSdkContent(<InteractiveDashboard dashboardId={dashboardId} />, {
-          theme: { components: { popover: { zIndex: zIndexTestCase.input } } },
+          sdkProviderProps: {
+            theme: {
+              components: { popover: { zIndex: zIndexTestCase.input } },
+            },
+          },
```

Comment:
We made the second argument of `mountSdkContent` accept a couple more flags, so we now need to pass the sdk provider's props to `sdkProviderProps`